src/components/BoardContainer.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import BoardSection from './BoardSection'
 
-const BoardContainer = ({ todo, inprogress, done, addWork }) => {
+const BoardContainer = ({ todo, inprogress, done, id }) => {
 
   return (
     <div id='boardcontainer'>
-      <BoardSection title={'할 일'} data={todo} $addWork={true} plus={addWork} />
+      <BoardSection title={'할 일'} data={todo} $addWork={true} id={id} />
```

Comment:
[Question]
boolean 변수에 $prefix 보다는 is_ prefix를 사용하면 좋을거 같고 true를 props로 넣고 싶으면 값을 생략할 수 있어요. $ prefix를 사용해야 하는 이유가 따로 있나요?
```suggestion
      <BoardSection title={'할 일'} data={todo} isAddWork id={id} />
```

Reply:
style components를 사용했을 때 동적 스타일 사용할 때 습관인 것 같아요..
동적 스타일 관련한 것도 아닌데 사용을 했네요..

Reply:
isAddWork 적용했습니다.
[commit-ea74a81]